lib/bemxjst/index.js

```diff
@@ -426,13 +426,7 @@ BEMXJST.prototype.tryRun = function(context, ent) {
   try {
     return ent.run(context);
   } catch (e) {
-    console.error('BEMXJST ERROR: cannot render ' +
-      [
-        'block ' + context.block,
-        'elem ' + context.elem,
-        'mods ' + JSON.stringify(context.mods),
-        'elemMods ' + JSON.stringify(context.elemMods)
-      ].join(', '), e);
+    context.onError(context, e);
```

Comment:
We decided with @sbmaxx that `return content.onError` could be better. ;-(

Reply:
@zxqfox then it will be not clear that onError returns value than will be used in templating result.

Why we need it? `onError()` is function for logging not for templating…

Reply:
@zxqfox I can publish patch if it’s important for you.

Reply:
Yes, probably naming troubles. But it should return something. It's very useful in some cases like debuggability in dev mode.

Reply:
@zxqfox okay, I going to publish patch right now…

Reply:
@zxqfox + bem-xjst@8.8.1

Reply:
@miripiruni <3 thank you!